.gitlab-ci.yml

```
@@ -137,12 +137,12 @@ variables:
   S3_DSD6_URI: s3://dsd6-staging
   RELEASE_VERSION_6: nightly
   RELEASE_VERSION_7: nightly-a7
-  DATADOG_AGENT_BUILDIMAGES: v10342924-8b0106f
-  DATADOG_AGENT_BUILDERS: v9930706-ef9d493
-  DATADOG_AGENT_WINBUILDIMAGES: v10342924-8b0106f
-  DATADOG_AGENT_ARMBUILDIMAGES: v10342924-8b0106f
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v10342924-8b0106f
-  DATADOG_AGENT_NIKOS_BUILDIMAGES: v10342924-8b0106f
+  DATADOG_AGENT_BUILDIMAGES: v10465962-8e9431f 
+  DATADOG_AGENT_BUILDERS: v10465962-8e9431f 
```

```suggestion
  DATADOG_AGENT_BUILDERS: v9930706-ef9d493 
```

This is the tag for builders that don't come from the `datadog-agent-buildimages` repository, so this variable shouldn't be updated to the same value.